rootfs/etc/cont-init.d/03-config.sh

```diff
@@ -24,6 +24,8 @@ RT_LOG_LEVEL=${RT_LOG_LEVEL:-info}
 RT_LOG_EXECUTE=${RT_LOG_EXECUTE:-false}
 RT_LOG_XMLRPC=${RT_LOG_XMLRPC:-false}
 
+NGINX_ACCESS_LOG_ENABLED=${NGINX_ACCESS_LOG_ENABLED:-true}
```

Comment:
Don't think that's the right place for this option. Should be in [General section](https://github.com/crazy-max/docker-rtorrent-rutorrent#general) under `LOG_IP_VAR` and maybe named `LOG_ACCESS`.